datafusion-examples/Cargo.toml

```diff
@@ -73,6 +73,7 @@ futures = { workspace = true }
 log = { workspace = true }
 mimalloc = { version = "0.1", default-features = false }
 object_store = { workspace = true, features = ["aws", "http"] }
+parquet-key-management = { version = "0.3", features = ["_test_utils"] }
```

Comment:
I am a little worried at taking a new dependency, even though I realize it is only for the examples -- the new dependency makes it that more effort to keep dependencies up to date, for example

It looks like this package is fairly new and has only 2 contributors https://github.com/G-Research/parquet-key-management-rs (I realize @adamreeve  you are one of them )